### PR TITLE
feat: add ability to copy playwright config into checkly config

### PR DESCRIPTION
**Pull request by ferrandiaz**
I hereby confirm that I followed the code guidelines found at [engineering guidelines](https://www.notion.so/checkly/Engineering-Guidelines-a3a165a203a04dc1a112f0e26b2f2d3f)

## Affected Components
* [x] CLI
* [x] Create CLI
* [ ] Test
* [ ] Docs
* [ ] Examples
* [ ] Other

## Notes for the Reviewer
Since create-cli and cli are 2 different workspaces I had to duplicate quite some code, if this extends more it would probably be wise to add a shared workspace or package between the 2 for the playwright config.

PR is separated in 2 commits.
1. Create CLI: adds option once creating the CLI to also import the playwright config file if found
2. CLI: `checkly sync-playwright` currently hidden, if passed adds or updates the playwright config of the checkly config.

### CODE REVIEW

**Review by umutuzgur — DISMISSED** [review]
LGTM

**Comment by clample** [comment]
Looks good to me 👍  I think that it could make sense to double check / tweak some of the user facing messages, though.

File: packages/cli/src/commands/sync-playwright.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    const pwtConfig = new PlaywrightConfigTemplate(config).getConfigTemplate()
+    const pwtConfigAst = this.findPropertyByName(recast.parse(pwtConfig), 'playwrightConfig')
+    this.addOrReplacePlaywrightConfig(browserCheckAst.value, pwtConfigAst)
+
+    const checklyConfigData = recast.print(checklyAst, { tabWidth: 2 }).code
+    const dir = path.resolve(path.dirname(configFile.fileName))
+    this.reWriteChecklyConfigFile(checklyConfigData, configFile.fileName, dir)
+
+    ux.action.stop('✅ ')
+    this.log('Successfully sync')
```
[the remark this answers is not in the document]
Comment (tnolet) [reply packages/cli/src/commands/sync-playwright.ts:47]: @clample "...updated the Checkly config file"

[the remark this answers is not in the document]
Comment (clample) [reply packages/cli/src/commands/sync-playwright.ts:47]: Good point, thanks 😅

File: packages/cli/src/commands/sync-playwright.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+import path from 'path'
+import { ux } from '@oclif/core'
+import PlaywrightConfigTemplate from '../playwright/playwright-config-template'
+
+export default class SyncPlaywright extends BaseCommand {
+  static hidden = true
+  static description = 'Sync playwright config'
+
+  async run (): Promise<void> {
+    ux.action.start('Sync playwright config with checkly config', undefined, { stdout: true })
```
[the remark this answers is not in the document]
Comment (ferrandiaz) [reply packages/cli/src/commands/sync-playwright.ts:15]: @clample Syncing or Synching? 🤔

[the remark this answers is not in the document]
Comment (clample) [reply packages/cli/src/commands/sync-playwright.ts:15]: Hmm, I think  Syncing is good. Apparently some British folks use Synching though. https://english.stackexchange.com/questions/681/synced-or-synched

**Review by umutuzgur — APPROVED** [review]

**Review by clample — APPROVED** [review]

**Pull request merged**